src/standalone.js

```diff
@@ -21,7 +21,27 @@ try {
   window.RUM_BASE = window.RUM_BASE || scriptSrc;
   window.RUM_PARAMS = window.RUM_PARAMS || scriptParams;
 
-  sampleRUM();
+  let status;
+  if (window.performance && window.performance.getEntriesByType) {
+    for (const { name, responseStatus, serverTiming } of window.performance.getEntriesByType("navigation")) {
```

Comment:
The `for... of` notation is used nowhere else in the project. Not saying it should not, it just makes this code different than the rest. Could have been a valid use if a `break` would be used but it is not the case.

Reply:
I committed @ekremney suggestion which removes the `for...of` as well.